vimeo-networking/src/main/java/com/vimeo/networking/model/VideoFile.java

```diff
@@ -63,18 +64,12 @@ public String toString() {
         }
     }
 
-    @Deprecated
-    @UseStag
     public enum VideoQuality {
```

Comment:
I actually think `MimeType` should be within the `ProgressiveVideoFile` file. I'd vote for `VideoQuality` to stay within this file. They are small enums and aren't used by multiple model types as represented by the API responses.